auslib/web/public/client.py

```diff
@@ -68,7 +68,10 @@ def getCleanQueryFromURL(url):
         if len(force_split) < 2:
             force_split = force_value.split('%3F', 1)
 
-        query['force'] = int(force_split[0])
+        if force_split[0]:
+            query['force'] = 1
+        else:
+            query['force'] = 0
```

Comment:
I don't think this is the right place to do this, actually. getCleanQueryFromURL merely makes the query parseable by fixing some known bad strings that get sent. I think the right place for this fix is at https://github.com/mozilla/balrog/blob/4d6f673a850cbd2924eb55145f047fadfceabeea/auslib/web/public/client.py#L92.

You should add some test cases to https://github.com/mozilla/balrog/blob/master/auslib/test/web/test_client.py to be sure, though. Some values we've seen for force are:
- nothing
- '1" name="Firefox 54.0" isOSUpdate="false" isCompleteUpdate="false" installDate="1498012260998" displayVersion="54.0" channel="release" buildID="20170608105825" appVersion="54.0"'
- '1)'
- '"|sleep	7	#'

Having test cases for those should be enough to ensure this works.